src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  user: User | {};
```

Comment:
The `user` property is typed as `User | {}`, which allows it to be an empty object. This could lead to runtime errors if the rest of the application expects a `User` object. Consider using `User | null` or `User | undefined` if the user might not be available, and handle these cases explicitly in your code.